packages/block-editor/src/components/inserter/menu.js

```diff
@@ -82,6 +82,8 @@ function InserterMenu(
 		if ( isZoomOutMode ) {
 			return 'patterns';
 		}
+
+		return 'blocks';
```

Comment:
Not sure why we missed this. The `getInitialTab` should always return value.

Reply:
Yeah, I'm surprised we missed it too. I guess we know we're human :)